src/components/AvatarGroup/docs/index.story.tsx

```diff
@@ -0,0 +1,83 @@
+import { AvatarGroup } from '../';
+import * as examples from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { AvatarGroupSize } from '../AvatarGroup';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+const items = [
+  {},
+  { name: 'anonymous' },
+  { name: 'Eve', src: 'https://randomuser.me/api/portraits/women/87.jpg' },
+  { name: 'John', src: 'https://randomuser.me/api/portraits/men/69.jpg' },
+  {},
+  { name: 'anonymous' },
+  { name: 'Eve', src: 'https://randomuser.me/api/portraits/women/87.jpg' },
+  { name: 'John', src: 'https://randomuser.me/api/portraits/men/69.jpg' },
+];
+
+const exampleItems = [
+  { label: 'few items', value: items.slice(0, 4) },
+  { label: 'many items', value: items },
+];
+
+export default {
+  category: 'Components',
+  storyName: 'AvatarGroup',
+  component: AvatarGroup,
+  componentPath: '../AvatarGroup.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-AvatarGroup',
+    items: exampleItems[1].value,
+    maxAmount: 5,
+    size: AvatarGroupSize.medium,
+  }),
+  exampleProps: {
+    items: exampleItems,
+    size: Object.keys(AvatarGroupSize),
+  },
+  dataHook: 'storybook-AvatarGroup',
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          importExample({
+            source: examples.importExample,
+          }),
+
+          divider(),
+
+          title('Examples'),
+
+          ...Object.keys(AvatarGroupSize)
+            .map(size => ({
+              title: size,
+              source: examples.example[size],
+            }))
+            .map(code),
+        ],
+      }),
+
+      ...[
+        { title: 'API', sections: [api()] },
+        { title: 'TestKit', sections: [testkit()] },
+        { title: 'Playground', sections: [playground()] },
+      ].map(tab),
+    ]),
+  ],
```

Comment:
Please add a `SettingsPanel` tab.
You can see an example for it in the `Card`'s storybook `index` file.
It will let us play with the border color, as if it was in the editor.

Reply:
Added settings panel here: https://github.com/wix/wix-ui-tpa/pull/138/commits/d9f4fedba9cbc3539c226ff988d0c8a9c6eba314

Trying to understand why component has some css class duplicated and looses stacking effect https://wix-wix-ui-tpa-pr-138.surge.sh/?path=///story/components--avatargroup

Probably, something wrong with Stylable setup in AvatarGroup component.